src/librustdoc/clean/types.rs

```diff
@@ -715,6 +715,7 @@ impl ItemKind {
             StructItem(s) => s.fields.iter(),
             UnionItem(u) => u.fields.iter(),
             VariantItem(Variant::Struct(v)) => v.fields.iter(),
+            VariantItem(Variant::Tuple(v)) => v.iter(),
```

Comment:
@camelid: I had to use a `Vec<Item>` because of this function.

Reply:
Ok, that seems good enough for now 👍